Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,6 +541,35 @@ bool isFunctionJson(string fullName)
             }
         }
 
+        private async Task LocalAppFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
+        {
+            var zipDeploymentInfo = (ZipDeploymentInfo)deploymentInfo;
+            var content = !string.IsNullOrEmpty(zipDeploymentInfo.ZipURL)
```

Comment:
>var [](start = 12, length = 3)

Let's define this as:
HttpContent content = ...

instead of:
var content = ...

as the type is not obvious and may confuse the reader to believe that this is a string